packages/docs/docs/hosting/heroku.mdx

```diff
@@ -0,0 +1,84 @@
+# Heroku
+
+## Overview
+
+For EzBackend to be compatible with hosting on Heroku, there needs to be some additional configuration.
+
+The below configuration is for the recommended configuration on Heroku, `EzBackend` + `Postgres`. You will probably need to configure all the items below for EzBackend to work as expected
+
+## Port Binding
+
+### Fix
+
+In `app.start`
+
+```ts
+app.start({
+  backend: {
+    listen: {
+      address: '0.0.0.0',
+    },
+  },
+});
+```
+
+### Explanation
+
+By default, EzBackend will only listen on address `127.0.0.1`. Since Heroku assigns a random-ish port to each instance, it will throw the error
+
+```
+Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch
+```
+
+To fix this, we need to listen on all addresses, or address `0.0.0.0`
+
```

Comment:
Can also add if heroku logs mention that : cannot find module 'typescript'
Move/copy from dev dependencies to dependencies

Reply:
I changed the build script instead, because using `ts-node-dev` in production is not recommended (Or at least requires additional configuration)